hammerdb-wrapper/hammerdb-wrapper.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python
+
+import argparse
+import sys
+import subprocess
+import os
+import re
+import elasticsearch
+
+
+def _run_hammerdb():
+    cmd = "cd /hammer; ./hammerdbcli auto /workload/tpcc-workload.tcl"
+    #cmd = "cd ~/Downloads/hammer/HammerDB-3.2; ./hammerdbcli auto workload.tcl"
+    process = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
+    stdout,stderr = process.communicate()
+    return stdout.strip(), process.returncode
+
+def _fake_run():
+    with open("hammerdb.log", "r") as input:
+        stdout = input.read()
+    return stdout,0
+
+def _parse_stdout(stdout):
+    data = []
+    iteration = 0
```

Comment:
Where is iteration being used in this function?

Reply:
@mkarg75

Reply:
It's not used at all - hammerdb itself has a parameter called iteration which I originally wanted to use but turned out to be not what its name suggests. I'm in touch with the hammerdb author and maintainer to clarify a few gaps I have in understanding some of the available settings. Will clean out whatever is no longer needed in the code, once that's done.